.bazelrc

```diff
@@ -143,6 +143,11 @@ build:mkl --define=tensorflow_mkldnn_contraction_kernel=0
 build:mkl --define=build_with_mkl_dnn_v1_only=true
 build:mkl -c opt
 
+build:mkl_threadpool --define=build_with_mkl=true --define=enable_mkl=true
```

Comment:
Please add a comment explaining what this config does (oneDNN using Eigen threadpool instead of OpenMP by default).